docs/development/contributing_guide.md

```diff
@@ -170,6 +170,53 @@ To increase the log level for the tests, set `SYNAPSE_TEST_LOG_LEVEL`:
 SYNAPSE_TEST_LOG_LEVEL=DEBUG trial tests
 ```
 
+### Running tests under PostgreSQL
+
+Invoking `trial` as above will use an in-memory SQLite database. This is great for
+quick development and testing. However, we only support using a PostgreSQL database
```

Comment:
I'm not aware that this is true, but perhaps:
```suggestion
quick development and testing. However, we recommend using a PostgreSQL database
```

Reply:
I had in mind https://matrix-org.github.io/synapse/latest/setup/installation.html?highlight=sqlite#using-postgresql , which says:

> SQLite is only acceptable for testing purposes. SQLite should not be used in a production server. Synapse will perform poorly when using SQLite, especially when participating in large rooms.

That said, still happy to make the change.

Reply:
I didn't know we said something quite so strong about it, but I would probably still make the change anyway as we would still support SQLite in the sense of making sure it doesn't just fall over and explode (even if it does run slowly, though I have to admit an old server of mine was on SQLite in the early days and it was good enough for what I was doing back then).